onyo/lib/commands.py

```diff
@@ -537,13 +537,14 @@ def onyo_mv(inventory: Inventory,
         subject = "mv"
         for s in sources:
             move_asset_or_dir(inventory, s, destination)
-    else:
-        # RENAME
+    elif len(sources) == 1 and inventory.repo.is_inventory_dir(sources[0]) and \
+            inventory.repo.is_inventory_dir(Path(*destination.parts[:-1])):
+        # RENAME directory
         subject = "ren"
-        if len(sources) != 1:
-            raise ValueError("Cannot rename multiple sources.")
-        else:
-            inventory.rename_directory(sources[0], destination)
+        inventory.rename_directory(sources[0], destination)
+    else:
+        raise ValueError("'onyo mv' can only move into an existing directory",
+                         "rename a single directory.")
```

Comment:
A caller knows where this is coming from and `'onyo mv'` would be command line specific. No need to name it, though. It's either being spit out by a call to `onyo mv` via CLI or raised from `onyo_mv()` when called via python interface. Either way, there's no need to mention its (context-specific) name.

```suggestion
        raise ValueError("Can only move into an existing directory",
                         " or rename a single directory.")
```